### ISSUE TITLE
Reading: How to Adapt Your Pretrained Multilingual Model to 1600 Languages

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#193]
## 0. Paper
@inproceedings{ebrahimi-kann-2021-adapt,
    title = "How to Adapt Your Pretrained Multilingual Model to 1600 Languages",
    author = "Ebrahimi, Abteen  and
      Kann, Katharina",
    booktitle = "Proceedings of the 59th Annual Meeting of the Association for Computational Linguistics and the 11th International Joint Conference on Natural Language Processing (Volume 1: Long Papers)",
    month = aug,
    year = "2021",
    address = "Online",
    publisher = "Association for Computational Linguistics",
    url = "https://aclanthology.org/2021.acl-long.351",
    doi = "10.18653/v1/2021.acl-long.351",
    pages = "4555--4567",
    abstract = "Pretrained multilingual models (PMMs) enable zero-shot learning via cross-lingual transfer, performing best for languages seen during pretraining. While methods exist to improve performance for unseen languages, they have almost exclusively been evaluated using amounts of raw text only available for a small fraction of the world{'}s languages. In this paper, we evaluate the performance of existing methods to adapt PMMs to new languages using a resource available for close to 1600 languages: the New Testament. This is challenging for two reasons: (1) the small corpus size, and (2) the narrow domain. While performance drops for all approaches, we surprisingly still see gains of up to 17.69{\%} accuracy for part-of-speech tagging and 6.29 F1 for NER on average over all languages as compared to XLM-R. Another unexpected finding is that continued pretraining, the simplest approach, performs best. Finally, we perform a case study to disentangle the effects of domain and size and to shed light on the influence of the finetuning source language.",
}

## 1. What is it?
They conducted zero-shot learning via cross-lingual settings.

## 2. What is amazing compared to previous works?
They use previous adaptation methods into low-resource languages.

### Zero-shot learning
- target: PoS tagging or NER in low-resource languages (XLM-R does not use)
- unlabeled data: the Bible in the target low-resource language
- labeled data: PoS or NER in English

### Domain Adaptation
- continued pre-training: train with a monolingual corpus in the target language (the Bible) 
   - **MLM**
   - **TLM**
   - MLM + TLM (**{M|T}LM**)
- vocabulary extension (**Extend**): train **a new** sentence piece model with extra 30,000 subwords
- adapter (**Adapters**): MLM model is fixed (frozen), and use a simple layer for adaptation

## 4. How did evaluate it?
In both tasks, pre-training improved the performance compared to Extend and Adapters.
<img width="487" alt="スクリーンショット 2021-08-03 14 55 57" src="https://user-images.githubusercontent.com/45454055/127965171-26ae3897-18af-470a-b070-d31b57dae6c7.png">
<img width="487" alt="スクリーンショット 2021-08-03 14 56 17" src="https://user-images.githubusercontent.com/45454055/127965204-b0654211-f576-4ca0-8acf-0e3d2793a09c.png">

## 5. Is there a discussion?
They use Russian, which is more similar to the target language than English.
Moreover, they use Wikipedia instead of the Bible.
<img width="487" alt="スクリーンショット 2021-08-03 15 01 05" src="https://user-images.githubusercontent.com/45454055/127965625-0477fc11-d7fd-4e52-a07c-f8914db6fbda.png">

### B-E→B-R (use more similar language)
XLM-R increased performance, but adaptation methods perform poorly.
They considered that **any performance gain from the unlabeled data is minimized** when the languages are similar.

### B-E→W-E (use different dataset)
Adaptation methods (MLM and Adapters) increased performance.

### B-E→W-R (use different dataset and more similar language)
MLM improved performance (+11.85).

<img width="487" alt="スクリーンショット 2021-08-03 14 56 36" src="https://user-images.githubusercontent.com/45454055/127965241-8b1b65b6-159d-4fe4-95fb-39c87d5bca52.png">
